bugs_and_glitches.md

```diff
@@ -326,7 +340,7 @@ There is a mistake in the AI logic for deciding which Pokémon for Sam to switch
 	inc a ; PLAY_AREA_BENCH_2
 ```
 
-**Fix:** Edit `AIDecide_PokemonTrader_PowerGenerator` in [src/engine/duel/ai/trainer_cards.asm](https://github.com/pret/poketcg/blob/master/src/engine/duel/ai/trainer_cards.asm):
+**Fix:** Edit `AIDecide_PokemonTrader_PowerGenerator` in [src/engine/duel/ai/decks/sams_practice.asm](https://github.com/pret/poketcg/blob/master/src/engine/duel/ai/decks/sams_practice.asm):
```

Comment:
Thanks for catching this. It looks like `AIDecide_PokemonTrader_PowerGenerator` needs to be changed to `AIPerformScriptedTurn` too.